### ISSUE TITLE
Time to switch to React🔥🔥

### ISSUE THREAD

**Issue by codewitgabi** [codewitgabi/django_quiz_app#13]
For a better user experience, I think react will be better just to prevent slow navigation when there's network issue faced by a user. @siufai6 what do you think?

**Comment by siufai6** [comment]
I don't understand React enough to comment.  But given the project complexity is not high now, it might be a good experiment at this point.

**Comment by codewitgabi** [comment]
Exactly. So would you mind working on the backend then while I focus on the frontend? All that there is is to build the APIs and we are done.

**Comment by siufai6** [comment]
yup. can try
